Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:16.14.0
+WORKDIR /chase-car-dashboard
+COPY . .
+COPY ["package.json", "package-lock.json*", "./"]
```

Comment:
The whole project is copied on the previous line. Does this line need to stay? If not, it should be deleted

Reply:
You are right just removed the second COPY line